packages/babel-runtime-corejs2/helpers/esm/iterableToArray.js

```diff
@@ -1,6 +1,6 @@
-import _Array$from from "../../core-js/array/from";
-import _isIterable from "../../core-js/is-iterable";
-import _Symbol from "../../core-js/symbol";
+import _Array$from from "@babel/runtime-corejs2/core-js/array/from";
+import _isIterable from "@babel/runtime-corejs2/core-js/is-iterable";
+import _Symbol from "@babel/runtime-corejs2/core-js/symbol";
```

Comment:
For users using PnP, is it allowed for a package to import itself?

Reply:
Yarn PnP does not support esm yet: https://github.com/yarnpkg/berry/issues/638

@merceyz Does PnP support `require("package/itself")`?

Reply:
Yes, that's totally fine